frontend/src/__tests__/cypress/cypress/tests/mocked/application.cy.ts

```diff
@@ -113,4 +114,28 @@ describe('Application', () => {
     aboutDialog.findText().should('contain.text', 'OpenShift');
     aboutDialog.findProductName().should('contain.text', 'OpenShift AI');
   });
+  it('should show the login modal when receiving a 403 status code', () => {
+    // Mock the intercept to return a 403 status code
+    cy.interceptOdh('GET /api/config', {
+      statusCode: 403,
+    }).as('getData403');
+
+    // Set up the sign-out intercept before visiting the page
+    cy.intercept('GET', '/oauth/sign_out').as('signOut');
```

Comment:
It would be good to add a typed interface function to the `interceptOdh` method. We tend to do this for all our well known URLs.